### DISCUSSION TITLE
Does wlc support self-sign certificate?

### DISCUSSION THREAD

**Discussion by razor313** [WeblateOrg/weblate#11099] [General]
Due to having a security policy, we aren't allowed to use wlc on HTTP (port 80) protocol, one possible and quick solution for us is using a self-sign certificate. so my question is does wlc support self-sign certificates?

**Comment by nijel** [comment]
You can specify it via REQUESTS_CA_BUNDLE environment variable, see https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification
